pkg/simulation/simulation.go

```diff
@@ -18,12 +20,13 @@ func NewSimulation(city *city.City) Simulation {
 func (s *Simulation) ResetTrams() {
 	s.trams = make([]*tram, len(s.city.GetTramTrips()))
 	for i, trip := range s.city.GetTramTrips() {
-		s.trams[i] = newTram(i, &trip)
+		s.trams[i] = newTram(i, &trip, &s.controlCenter)
 	}
 }
 
 func (s *Simulation) FetchData(url string) {
 	s.city.FetchCityData(url)
+	s.controlCenter = controlcenter.CreateControlCenter(s.city)
```

Comment:
I think we missed it in the last review, it should be `NewControlCenter` according to convention.
```suggestion
	s.controlCenter = controlcenter.NewControlCenter(s.city)
```

Reply:
Applied